view/utils/init-table-sorting-on-client.js

```diff
@@ -16,6 +16,23 @@ module.exports = function (domElement/*, opts */) {
 		// set type, either numeric or text 
 		type: 'numeric'
 	});
+	window.jQuery.tablesorter.addParser({
+		// set a unique id 
+		id: 'dates',
+		is: function (s) {
+			// return false so this parser is not auto detected 
```

Comment:
While stating that returning false will prevent auto-detection is valuable information I would like to know why we don't want it to be auto-detected as well...